sssd_test_framework/utils/sss_override.py

```diff
@@ -0,0 +1,402 @@
+        :type certificate: str | None, optional
+        :return: Self.
+        :rtype: SSSOverrideUser
+        """
+        args: CLIBuilderArgs = {
+            "name": (self.util.cli.option.VALUE, name),
+            "uid": (self.util.cli.option.VALUE, uid),
+            "gid": (self.util.cli.option.VALUE, gid),
+            "home": (self.util.cli.option.VALUE, home),
+            "gecos": (self.util.cli.option.VALUE, gecos),
+            "shell": (self.util.cli.option.VALUE, shell),
+            "certificate": (self.util.cli.option.VALUE, certificate),
+        }
+
+        self.util.logger.info(f"Creating local override for user {self.user} on {self.util.host.hostname}")
+        self.util.host.ssh.exec(["sss_override", "user-add", self.user] + self.util.cli.args(args))
+
+        return self
+
+    def modify(
+        self,
+        *,
+        uid: int | None = None,
+        gid: int | None = None,
+        name: str | None = None,
+        home: str | None = None,
+        gecos: str | None = None,
+        shell: str | None = None,
+        certificate: str | None = None,
+    ) -> SSSOverrideUser:
+        """
+        Modify existing local override for user. Parameters that are not set are ignored.
+
+        :param uid: User id, defaults to None
+        :type uid: int | None, optional
+        :param gid: Primary group id, defaults to None
+        :type gid: int | None, optional
+        :param name: Name of user id
+        :type name: str | None, optional
+        :param home: Home directory, defaults to None
+        :type home: str | None, optional
+        :param gecos: GECOS, defaults to None
+        :type gecos: str | None, optional
+        :param shell: Login shell, defaults to None
+        :type shell: str | None, optional
+        :param certificate: certificate, defaults to None
+        :type certificate: str | None, optional
+        :return: Self.
+        :rtype: SSSOverrideUser
+        """
+
+        args: CLIBuilderArgs = {
+            "name": (self.util.cli.option.VALUE, name),
+            "uid": (self.util.cli.option.VALUE, uid),
+            "gid": (self.util.cli.option.VALUE, gid),
+            "home": (self.util.cli.option.VALUE, home),
+            "gecos": (self.util.cli.option.VALUE, gecos),
+            "shell": (self.util.cli.option.VALUE, shell),
+            "certificate": (self.util.cli.option.VALUE, certificate),
+        }
+
+        self.util.logger.info(f"Modifying local override for user {self.user} on {self.util.host.hostname}")
+        self.util.host.ssh.exec(["sss_override", "user-add", self.user] + self.util.cli.args(args))
+
+        return self
+
+    def delete(self) -> SSSOverrideUser:
+        """
+        Delete the local override for user.
+        """
+        self.util.logger.info(f"Deleting local override for user {self.user} on {self.util.host.hostname}")
+        self.util.host.ssh.exec(["sss_override", "user-del", self.user])
+
+        return self
+
+    def backup(
```

Comment:
Move this to SSSOverrideUtils since it does not operate on single user and rename it to export since this is the operation that is called on sss_override to avoid confusion. Same for restore and groups.

Reply:
Since import cannot be used as name lets go with export_data and import_data.